Bottom: 5cb88be403c89a5ec86cc7f01f955097f4522cd5
Top:    39f80103c7ed1e6dd3ba3b39a722d5650e408a13
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-12 14:05:40 -0500

Refresh of gitlab-auth

---
diff --git a/docker/buildbot-master/Dockerfile b/docker/buildbot-master/Dockerfile
index 1baa3f49..a79778ef 100644
--- a/docker/buildbot-master/Dockerfile
+++ b/docker/buildbot-master/Dockerfile
@@ -18,7 +18,8 @@ RUN pip --no-cache-dir install \
     'buildbot-console-view' \
     'buildbot-grid-view' \
     'buildbot-waterfall-view' \
-    'buildbot-www'
+    'buildbot-www' \
+    'ldap3'
 
 # Create buildbot user
 ARG BUILDBOT_UID=1000
